scopesim_templates/extragalactic/exgal_models.py

```diff
@@ -29,10 +27,9 @@ class VelField(Fittable2DModel):
     ----------
     ellip : float, u.Quantity
         Ellipticity on the sky
-
     theta : float, u.Quantity
-        Position angle of the major axis wrt to north (=up) measured
-        counterclockwise.
+        Position angle of the major axis. The rotation angle increases
+        counterclockwise from the positive x axis.
```

Comment:
I copied this from the description of `theta` in https://docs.astropy.org/en/stable/api/astropy.modeling.functional_models.Sersic2D.html, which is what's used for the intensity profile. We should probably have a different (more sky-oriented) definition of the angle at the top level and then convert it to whatever astropy used. But I do think we should absolutely not mix the internal definitions of these angles. That is, our own velocity profiles should have the same definition of these things as astropy's Sersic profile.

Reply:
The astropy documentation is confusing as well then :-). But at least you have to give x and y yourself, as the example says

```
mod = Sersic2D(amplitude=1, r_eff=25, n=4, x_0=50, y_0=50,
               ellip=0.5, theta=-1)
img = mod(x, y)
```

So you can kinda infer from that that the value `x_0` is related to the first argument of `mod()`, and `y_0` to the second argument, and the user has full control over what the x and y direction mean based on that. But without the example, it would have been impossible to tell what x and y are. The picture also clarifies what counterclockwise means, but otherwise you'd have to guess.

But with ScopeSim "the x-direction" is not something that is under the control of the user, so it is unclear what is meant. And I'm not sure whether ScopeSim uses the same convention about which directory is x and which is y.

But if you say it is internally more consistent now, then I believe you. I just wanted to note that it is not obvious. Maybe the notebook wil clarify, just as with astropy.

Reply:
> Maybe the notebook wil clarify, just as with astropy.

Indeed it should. I tested it with different values for `theta` from 0 upwards, plus a significant ellipticity to see where the SMA is, and (using the ol' `origin="lower"` ofc) the imshow plot of the simulation readout HDU data shows a horizontal ellipse for `theta=0` that turns counterclockwise towards the vertical when `theta` is increased. This matches also the plot of the input source (field) data, as well as my interpretation of astropy's docstring. So I'm rather confident that things are consistent here :)